server/game/gameSystems/LookMoverDeckCardsTopOrBottomSystem.ts

```diff
@@ -0,0 +1,81 @@
+import type { AbilityContext } from '../core/ability/AbilityContext';
+import type { Card } from '../core/card/Card';
+import { CardTargetSystem, type ICardTargetSystemProperties } from '../core/gameSystem/CardTargetSystem';
+import { GameEvent } from '../core/event/GameEvent';
+import { Location } from '../core/Constants';
+import { lookAt, moveCard } from './GameSystemLibrary';
+
+export interface ILookMoveDeckCardsTopOrBottomProperties extends ICardTargetSystemProperties {
+    amount: number;
+}
+
+export class LookMoveDeckCardsTopOrBottomSystem<TContext extends AbilityContext = AbilityContext> extends CardTargetSystem<TContext, ILookMoveDeckCardsTopOrBottomProperties> {
+    public override readonly name = 'LookMoveDeckCardsTopOrBottomSystem';
+
+    protected override defaultProperties: ILookMoveDeckCardsTopOrBottomProperties = {
+        amount: 1,
+    };
+
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    public override eventHandler(event): void { }
+
+    public override queueGenerateEventGameSteps(events: GameEvent[], context: TContext): void {
+        const player = context.player;
+        const { amount } = this.generatePropertiesFromContext(context);
+        const deckLength = player.drawDeck.length;
+        if (deckLength === 1) {
+            events.push(lookAt({
```

Comment:
We want to try and avoid importing from `GameSystemLibrary` in any GameSystem files because it creates import loops that could cause issues eventually. Let's remove the `lookAt()` import and instead use `new LookAtSystem()`, same for moveCard.

Reply:
changed to new LookAtSystem() and new MoveCardSystem